cmd/ocsp-responder/main.go

```diff
@@ -79,19 +79,36 @@ func (src *DBSource) Response(req *ocsp.Request) ([]byte, bool) {
 	serialString := core.SerialToString(req.SerialNumber)
 	log.Debug(fmt.Sprintf("Searching for OCSP issued by us for serial %s", serialString))
 
-	var ocspResponse core.OCSPResponse
-	// Note: we order by id rather than createdAt, because otherwise we sometimes
-	// get the wrong result if a certificate is revoked in the same second as its
-	// last update (e.g. client issues and instant revokes).
-	err := src.dbMap.SelectOne(&ocspResponse, "SELECT * from ocspResponses WHERE serial = :serial ORDER BY id DESC LIMIT 1;",
-		map[string]interface{}{"serial": serialString})
+	var response []byte
+	defer func() {
+		if len(response) != 0 {
+			log.Info(fmt.Sprintf("OCSP Response sent for CA=%s, Serial=%s", hex.EncodeToString(src.caKeyHash), serialString))
+		}
+	}()
+	// Note: we first check for an OCSP response in the certificateStatus table (
+	// the new method) if we don't find a response there we instead look in the
+	// ocspResponses table (the old method) while transitioning between the two
+	// tables.
+	err := src.dbMap.SelectOne(
+		&response,
+		"SELECT ocspResponse FROM certificateStatus WHERE serial = :serial",
```

Comment:
The current schema doc say to name something as the plural of what it is. I know you're avoiding that because we already have a plural, but I think "lastOCSPResponses" would be fine.